src/implementations/twig/components/form-group/form-group.html.twig

```diff
@@ -36,15 +40,26 @@
 {% set _input = input|default({}) %}
 {% set _width = _input.width|default('') %}
 {% set _id = _input.id|default('') %}
+{% set _group_id = _input.group_id|default('') %}
```

Comment:
mmh..but why is this parameter expected in the input and not in the root here? It's hard to expect users to really use this, but it doesn't have much to do it with the rendering of a form input field, it is actually completely unused in these templates so if feels to me that this parameter should belong to the form group only and be passed when needed to the form input templates internally.